### ISSUE TITLE
Simplify orthographic camera parameters

### ISSUE THREAD

**Issue by andy-sweet** [chanzuckerberg/idetik#89]
The orthographic camera currently allows us to specify left, right, bottom, top coordinates as well as the affine transform it gets from being a `RenderableObject`. These parameters seem to out-number the effective degrees of freedom it has in terms of the transforms it produces.

The goal of this issue is to reduce those parameters.

For example, a complete set of parameters for an orthographic camera could be a position, an up-vector, and a positive width and height.

As part of this issue, we may need to ensure that the perspective camera is as consistent with the orthographic camera as possible. For example, if the orthographic camera is no longer a renderable object with an `AffineTransform` (and instead has a position and up-vector), then the perspective camera should be similar.

**Comment by andy-sweet** [comment]
As part of this issue, it would be good to modify at least one of the examples to show how to modify some of these new more intuitive parameters (e.g. pan and zoom). We may want to provide helpers to make these easy and concise in the examples.

**Comment by aganders3** [comment]
> For example, a complete set of parameters for an orthographic camera could be a position, an up-vector, and a positive width and height.

This is true but I think it's also okay to have more than one way to specify this. In different applications it may be more intuitive to set parameters in a certain way.

> For example, if the orthographic camera is no longer a renderable object with an AffineTransform (and instead has a position and up-vector), then the perspective camera should be similar.

I don't have a strong preference on whether the camera remains a renderable object or not - this feels like a bigger question about how much to we protect clients from an attractive nuisance (modifying a `transform` property directly). For example a client application can already access/modify the transform on each renderable object, but I don't think we're encouraging it.

I think it would be useful for the `AffineTransform` to have some additional methods like `setScale`, `setRotation`, and `setTranslation`. Then cameras could have more intuitive methods/properties to `postion` and `up`, but ultimately they would just be more ergonomic wrappers for modifying transform and/or projection matrices. So you'd set `camera.positon` instead of calling `camera.transform.setTranslation`.

**Comment by aganders3** [comment]
Related post from @shlomnissan [on Slack](https://czi-sci.slack.com/archives/C07GHGSJSQP/p1748362561052899):
> I wanted to share a few thoughts on how I think camera controls should fit into the runtime, and get your perspective:
> 1. **Camera controls shouldn’t be owned by the renderer or embedded in camera classes.** Having the renderer manage controls creates unnecessary coupling, and embedding them in camera classes reduces flexibility—we’ll likely want different control types, or even allow consumers to plug in their own. My current thinking is that the runtime or layer manager should own top-level camera controls that affect all layers, while each layer manages its own layer-specific controls (e.g., picking).
> 2. **Controls should operate through the camera transform rather than modifying individual camera properties.** This keeps interactions like panning, zooming, and rotating consistent and composable, and avoids scattered updates to position, orientation, or zoom. This separation also improves clarity, simplifies matrix composition, and makes frustum calculations more predictable.
> 3. **Each control class should target a single camera type (orthographic or perspective).** The interaction logic is different enough that combining both paths in one class leads to unnecessary complexity and harder-to-maintain code. Keeping them separate makes the implementation clearer and more focused.
Let me know what you think. It’s possible I’m too anchored to how traditional scene graph libraries handle this, and that may not be the best fit here.
